oat-sa/extension-tao-itemqti
# Feature/ADF-1245/Add ES5 bundles

### Related to: https://oat-sa.atlassian.net/browse/ADF-1245

### Requires
- [ ] https://github.com/oat-sa/tao-core/pull/3655

### Summary

Add bundle tasks for generating an ES5 flavour of some bundles (taoQtiItemRunner)

### Details

The internal compiler has been updated, see the companion PR.
We can now have ES5 bundles aside from the regular ones. To properly use them, we need the full chain to declare dependencies to ES5. See the companion PRs for some examples.

### How to test
- checkout the branch
- install the dependencies: `cd tao/views/build && npm i`
- run bundle tasks: `cd tao/views/build && npx grunt taoqtiitembundle`
- check the frontend in production mode
